### ISSUE TITLE
CI (private) Failing after 45m —  Timeout is too short!

### ISSUE THREAD

**Issue by Jacob-Levy** [lowRISC/opentitan#10916]
When running without timeout it passes!
                               
00:18:48  [    build    ]: [Q: 000, D: 000, P: 041, F: 000, K: 000, T: 041] 100%                                                                
00:51:12  [     run     ]: [Q: 000, D: 000, P: 107, F: 000, K: 000, T: 107] 100%  

Can someone increase the timeout to 60m

**Comment by GregAC** [comment]
Timeout increased to 60 minutes. Will leave this open until we've observed reliable private ci runs again.

**Comment by rswarbrick** [comment]
Thanks for doing that, Greg. But have we seen widespread flakiness? Or is it just with #10725?

**Comment by GregAC** [comment]
I've seen a couple more timeouts. We can leave this value as is for now then once we've got a week or so of new run history look at actual times and decide to keep it or not.

**Comment by sriyerg** [comment]
Seems reasonably stable to me (aside from the timeouts, which is explained by the increase in number of tests). 
https://dev.azure.com/lowrisc/opentitan-private/_build?definitionId=8&_a=summary

**Comment by sriyerg** [comment]
The runtimes are pretty wide, ranging from 30 to 45 minutes. I will investigate if we are seeing a similar spread at our end (unlikely), My best guess is simulations are going into pending state waiting to acquire license, because licenses are getting consumed by other private CI runs happening in parallel due to other PRs. If you have access to the private CI artifacts, perhaps you could investigate the `run.log`s to see if there are "waiting to acquire license" messages?

We should consider increasing the number of licenses and running more in parallel. 
@domrizz0

**Comment by Jacob-Levy** [comment]
What happen if not egnouth VCS runtime licenses are not available?
I run into this issue localy and just recently increased the number of 
available licenses and it seem to resolve the issue...

Is there a way to inrease the timeout when not egnouth VCS runtime licenses are availabe?

**Comment by GregAC** [comment]
Just reviewed the timeout here, we're still on 90 minutes. Vast majority completed in under 50 minutes but some are taking over an hour. I've decided to leave the timeout as is. Whilst the slight variability in run time and long timeout isn't ideal I don't think it's been causing us any noticeable issues (private CI queuing generally isn't the factor slowing down CI, public CI is). Closing this issue now. Please open a new issue if there are any problems with the long timeout or the run-time variability.
